sourcecode/hub/resources/views/content/show.blade.php

```diff
@@ -1,27 +1,142 @@
+                        <x-icon name="info-circle" class="ms-2" />
+                    </span>
+                </h2>
                 <ul class="p-0">
-                    @foreach ($content->versions as $version)
-                        <li class="d-block p-1 mb-1 {{ $version->published ? ($content?->latestPublishedVersion->is($version) ? 'bg-success text-bg-success' : 'bg-success-subtle'): 'bg-danger-subtle' }}">
-                            {{ $version->created_at }}
-                            @if ($content?->latestPublishedVersion->is($version))
-                                <span class="d-block">{{ trans('messages.published') }}</span>
-                            @endif
+                    @foreach ($content->versions->sortByDesc('created_at')->take(3) as $index => $version)
+                        <li class="d-flex flex-column p-1 mb-1 rounded {{ $loop->first ? 'border border-success bg-success-subtle' : 'border border-secondary bg-white' }}">
+                            <div class="version-details-container d-flex w-100 align-items-center justify-content-between">
+                                <div class="version-details-80">
+                                    <div class="version-number">
+                                        <b>{{ trans('messages.version') }} {{ $index + 1 }}</b>
+                                    </div>
+                                    <div class="version-date">
+                                        {{ $version->created_at }}
+                                    </div>
+                                    <div class="version-status">
+                                        @if ($version->published)
+                                            {{ trans('messages.published') }}
+                                        @else
+                                            <span>{{ trans('messages.unpublished') }}</span>
+                                        @endif
+                                    </div>
+                                </div>
+                                @unless ($version->published)
+                                    <div class="rounded-pill bg-light d-flex justify-content-center align-items-center p-2 m-2 text-black fw-bold bg-light">Draft</div>
+                                @else
+                                    <div class="version-icons-20 d-flex justify-content-center align-items-center p-2 m-2">
+                                        <span class="text-black fw-bold"><x-icon name="check2-circle" class="text-black"/></span>
+                                    </div>
+                                @endunless
+                            </div>
                         </li>
                     @endforeach
                 </ul>
+
+                @if(count($content->versions) > 3)
+                    <button class="btn btn-link d-flex align-items-center" type="button" data-bs-toggle="collapse" data-bs-target="#collapseVersions" aria-expanded="false">
```

Comment:
`aria-label` is required, the only content here is a icon that is hidden to screen readers. The `icon` component adds `aria-hidden="true"` if no label is set

Reply:
Add a `aria-label` to the `button`